src/core/plugins/vuetify.js

```diff
@@ -86,6 +86,7 @@ import Vuetify, {
   VScrollYTransition,
   VSkeletonLoader,
   VChip,
+  VChipGroup,
```

Comment:
Is this necessary? doesn't seem like it's being used anywhere.

Reply:
Yeah....
That is for the mobile friendly tab buttons.
![Screenshot from 2022-07-28 21-04-01](https://user-images.githubusercontent.com/9893774/181680997-77e0ad21-4796-4137-a6cb-14bd0147fb0b.png)

Reply:
I already added VChipGroup to mew-components, but I still get an VChipGroup is missing error. After I add VChipGroup locally, it doesn't show the error.
https://github.com/MyEtherWallet/mew-components/blob/master/src/plugins/vuetify.js

![Screenshot from 2022-08-05 15-10-01](https://user-images.githubusercontent.com/9893774/183218085-375fa093-185b-46b5-b5b3-d01515cc07f7.png)